ol3-cesium-plugin/gakmlsynchronizer.js

```diff
@@ -0,0 +1,112 @@
+goog.provide('olcs.GaKmlSynchronizer');
+goog.require('olcs.AbstractSynchronizer');
+
+
+/**
+ * Unidirectionally synchronize geoadmin kml layers to Cesium.
+ * @param {!ol.Map} map
+ * @param {!Cesium.Scene} scene
+ * @param {!Cesium.DataSourceCollection} dataSources
+ * @constructor
+ * @extends {olcs.AbstractSynchronizer}
+ * @api
+ * @struct
+ */
+olcs.GaKmlSynchronizer = function(map, scene, dataSources) {
+
+  /**
+   * @protected
+   */
+  this.dataSources_ = dataSources;
+
+  goog.base(this, map, scene);
+};
+goog.inherits(olcs.GaKmlSynchronizer, olcs.AbstractSynchronizer);
+
+
+/**
+ * @inheritDoc
+ */
+olcs.GaKmlSynchronizer.prototype.createSingleLayerCounterparts =
+    function(olLayer) {
+
+  var dsP;
+  var factory = olcs.obj(olLayer)['getCesiumDataSource'];
+  
+  if (factory) {
+    dsP = factory(this.scene);
+  }
+
+  if (!dsP) {
+    /** @type {string} */
+    var url = olcs.obj(olLayer)['url'];
+    
+    if (!(olLayer instanceof ol.layer.Layer) || olLayer.get('type') != 'KML'
+        || !url || /:\/\/public\./.test(url)) {
+      return null;
+    }
+
+    /** @type {string|Document} */
+    var loadParam = url;
+
+    /** @type {string} */
+    var kml = olcs.obj(olLayer.getSource()).get('kmlString');
+    if (kml) {
+      loadParam =(new DOMParser()).parseFromString(kml, 'text/xml');
+    }
+    dsP = Cesium.KmlDataSource.load(loadParam, {
```

Comment:
Yes I found a solution for that. It's better because we transform urls in the KML when we parse it.